conda-recipe/meta.yaml

```diff
@@ -38,7 +38,7 @@ requirements:
         - pkg-config
         - python # requirements for validphys
         - reportengine >=0.30.25 # see https://github.com/NNPDF/reportengine
-        - matplotlib >=3.3.0
+        - matplotlib >=3.3.0,<3.8
```

Comment:
Please add an in-line comment pointing to this PR, or even open an issue

Reply:
```suggestion
        - matplotlib >=3.3.0,<3.8  # see https://github.com/NNPDF/nnpdf/pull/1809
```